quadriga/src/main/java/edu/asu/diging/quadriga/core/model/MappedTripleGroup.java

```diff
@@ -0,0 +1,71 @@
+package edu.asu.diging.quadriga.core.model;
+
+import javax.persistence.Id;
+
+import org.bson.types.ObjectId;
+
+/**
+ * A MappedTripleGroup is used to map Concepts and Predicates to a collection
```

Comment:
that's incorrect. A mapped triple group groups together all the mapped triples that belong to specific mappings (e.g. all the triples resulting from default mappings, or all the triples that results from a user specified set of mappings).